cmd/check.go

```diff
@@ -11,7 +13,7 @@ func setupCheckCommand() *cobra.Command {
 		Short: "Check the package",
 		Long:  "Use check command to verify if the package is correct in terms of formatting, validation and building.",
 		RunE: func(cmd *cobra.Command, args []string) error {
-			err := composeCommandActions(cmd, args,
+			err := internalCmd.ComposeCommandActions(cmd, args,
```

Comment:
nit: is it possible not to expose this method? or maybe put it in the `cmd/utils`?

Reply:
Sure, originally I was thinking of putting it in `cmd/utils.go` but I thought `cmd` was reserved (by convention) for the external API. That's why I put it under `internal/cmd/utils.go` instead.

Reply:
Hmm... I don't think it can shared with some other cmd packages. I think we don't need to artificially extract it to a separate package. It should be fine just not exposed in cmd.

Reply:
Okay, relocated in 7dba5c5. Please take a look to see if this is what you were expecting. Thanks!